packages/legend-application-studio/src/stores/graphModifier/DomainGraphModifierHelper.ts

```diff
@@ -335,6 +785,72 @@ export const association_changePropertyType = action(
   },
 );
 
+export const association_arrangeTaggedValues = action(
+  (
+    association: Association,
+    sourceTaggedValue: TaggedValue,
```

Comment:
I think you can leave these as `val1` and `val2` to make things look more succinct :)

Reply:
sounds good will do! :D